File: Python/Monitor.py
# monitor.py
from Peripheral import Peripheral

class Monitor(Peripheral):
    """Monitor peripheral class"""
    
    def __init__(self, ukuran_inch=0.0, refresh_rate=0, resolution="", merk="", nama="", harga=0,
                 tipe_peripheral="Monitor", koneksi="HDMI"):
        super().__init__(merk, nama, harga, tipe_peripheral, koneksi)
        self.ukuran_inch = ukuran_inch
        self.refresh_rate = refresh_rate
        self.resolution = resolution
    
    def get_ukuran_inch(self):
        return self.ukuran_inch
    
    def set_ukuran_inch(self, ukuran_inch):
        self.ukuran_inch = ukuran_inch
    
    def get_refresh_rate(self):
        return self.refresh_rate
    
    def set_refresh_rate(self, refresh_rate):
        self.refresh_rate = refresh_rate
    
    def get_resolution(self):
        return self.resolution
    
    def set_resolution(self, resolution):
        self.resolution = resolution
    
    def tampilkan_info(self):
        print(f"{self.tipe_peripheral}: {self.merk} {self.nama} ({self.ukuran_inch}', {self.resolution}, {self.refresh_rate} Hz)")
        print(f"    Koneksi: {self.koneksi}, Harga: Rp{self.harga}")

File: Java/Monitor.java
public class Monitor extends Peripheral {
    private float ukuranInch;
    private int refreshRate;
    private String resolution;
    
    public Monitor() {
        super();
        this.ukuranInch = 0.0f;
        this.refreshRate = 0;
        this.resolution = "";
    }
    
    public Monitor(float ukuranInch, int refreshRate, String resolution, 
                  String merk, String nama, double harga, 
                  String tipePeripheral, String koneksi) {
        super(merk, nama, harga, tipePeripheral, koneksi);
        this.ukuranInch = ukuranInch;
        this.refreshRate = refreshRate;
        this.resolution = resolution;
    }
    
    public Monitor(float ukuranInch, int refreshRate, String resolution, 
                  String merk, String nama, double harga) {
        this(ukuranInch, refreshRate, resolution, merk, nama, harga, "Monitor", "HDMI");
    }
    
    public float getUkuranInch() {
        return ukuranInch;
    }
    
    public void setUkuranInch(float ukuranInch) {
        this.ukuranInch = ukuranInch;
    }
    
    public int getRefreshRate() {
        return refreshRate;
    }
    
    public void setRefreshRate(int refreshRate) {
        this.refreshRate = refreshRate;
    }
    
    public String getResolution() {
        return resolution;
    }
    
    public void setResolution(String resolution) {
        this.resolution = resolution;
    }
    
    @Override
    public void tampilkanInfo() {
        System.out.println(tipePeripheral + ": " + merk + " " + nama + 
                           " (" + ukuranInch + "', " + resolution + ", " + refreshRate + " Hz)");
        System.out.println("    Koneksi: " + koneksi + ", Harga: Rp" + harga);
    }
}